.github/analyst_guide_data_validation.md

```diff
@@ -0,0 +1,171 @@
+# Data validation
+
+This section covers the topic of **how to validate Workplace Analytics data**. Before you apply any functions to your data set and can start looking for new insights, it is recommended that you perform data validation. This best practice is applicable regardless of whether your aim is to explore the data, to establish a baseline, or to perform advanced analytics.
+
+## Why validate?
+
+There are several reasons why you should validate your Workplace Analytics data.
+
+1. There may be gaps, anomalies, or errors in the organizational data, such as missing data or excessive / insufficient granularity. This may require rectifying at source, or the resulting data should be interpreted differently, e.g. any biases caveated or accounted for. 
+2. Outliers may exist in Workplace Analytics data, and often for very legitimate reasons. For instance, collaboration hours for a particular week or employee may be significantly low due to public holidays or paid time off. If these outliers are not surfaced and addressed accordingly, the interpretation of the data may be incorrect.
+
+In a nutshell, it is good practice to have a comprehensive understanding of the data context and checks for common biases, errors, and anomalies prior to analysis, as otherwise we would risk the quality and the reliability of the analysis. 
+
+## Know what you are looking at
+
+Before you begin with data validation, it's helpful to know what data set you are looking at, which includes information such as: 
+
+  - What query type is loaded
+  - Number of unique employees in the dataset
+  - Date range of the dataset
+  - Organizational attributes int he dataset
+
+This can all be done with `check_query()`:
+
+```R
+library(wpa)
+check_query(sq_data)
+```
+
+The resulting output would look something like the following:
```

Comment:
The resulting output would look something like the following and will give you a first insights into your data: